### DISCUSSION TITLE
Zero-U PDU Options in Netbox?

### DISCUSSION THREAD

**Discussion by zveroboy152** [netbox-community/netbox#5908] [Q&A] [answered]
Good day!

We're using Netbox for our Datacenter management, and to keep stuff in line.  It's working great, however as we're adding out PDUs to it I've realized there's no option (that I can see) for a Zero-U PDU.  Is this an option, or anything that can be done in netbox?

**Comment by jeremystretch** [comment, marked as answer]
Yep. Just set the U height to 0 when creating the device type in NetBox. 0U devices can be installed in racks, but cannot be assigned to a specific rack position.

![Screenshot_2021-03-02 Add a new device type - NetBox](https://user-images.githubusercontent.com/13487278/109710097-412b3580-7b6b-11eb-913c-e73907b7b74d.png)

**Reply by zveroboy152** [reply]
You're a king.  :-)  We appreciate this project, and all the hard work you, and your small team, do!

**Comment by chas0rde** [comment]
Appending on this discussion:
We have the similar think. Currently we just have no Unit configured for the actual devices (which then shows them as non-racked devices in the specific rack.
But we also utilize the cable-management space on one side of our 800mm racks to vertically mount e.g. fiber splice-boxes or cable managers (which are device-types with Heigth 1). Those also show non-racked
It would be nice to be able to have an option to position them as well (e.g. 47U Rack + 6 "Zero-Units" in our case) so it becomes clear where they are physically located in the rack (since this might be relevant for planning of cable length for example)
